frontends/web/src/app.tsx

```diff
@@ -71,27 +72,36 @@ class App extends Component<Props, State> {
     /**
      * Gets fired when the route changes.
      */
-    private handleRoute = () => {
+    private handleRoute = async (e:RouterOnChangeArgs) => {
         if (panelStore.state.activeSidebar) {
             toggleSidebar();
         }
+
+        await sleep(0); // this must be run async
+        const {accounts} = this.state;
+        const isIndex = e.url === '/' || e.url === '/index.html' || e.url === '/android_asset/web/index.html';
+        
+        if (isIndex && accounts.length > 0 ) {
+            route('/account-summary', true);
+            return;
+        }
     }
 
     public componentDidMount() {
         this.unsubscribe = apiWebsocket(({ type, data, meta }) => {
             switch (type) {
-            case 'backend':
-                switch (data) {
-                case 'newTxs':
-                    apiPost('notify-user', {
-                        text: this.props.t('notification.newTxs', {
-                            count: meta.count,
-                            accountName: meta.accountName,
-                        }),
-                    });
+                case 'backend':
```

Comment:
needless reformatting? it's best to stick to the current formatting and keep reformatting changes to separate dedicated commits/PRs.